test/unit/Hbar.js

```diff
@@ -18,4 +18,84 @@ describe("Hbar", function () {
 
         expect(tinybar).to.eql(hbar.toTinybars().toString());
     });
+
+    it("should reverse Hbar units using [to|from]String()", function () {
+        let hbar = new Hbar("100.00", HbarUnit.Hbar);
+        let toString = String;
+        let fromString = String;
```

Comment:
I'm not sure if this is the case, but you setting the value equal to `String` is not the same thing as defining the type of this variable. To define a variable of type `string | undefined` you'd do 
```javascript
/** @type {string=} */
let fromString;
```